templates/terraform/custom_import/self_link_as_name_set_project.go.erb

```diff
@@ -0,0 +1,18 @@
+
+	config := meta.(*Config)
+
+	// current import_formats can't import fields with forward slashes in their value
+	if err := parseImportId([]string{"(?P<name>.+)"}, d, config); err != nil {
+		return nil, err
+	}
+
+	stringParts := strings.Split(d.Get("name").(string), "/")
+	if len(stringParts) < 2 {
+		return nil, fmt.Errorf(
+				"Could not split project from name: %s",
+				d.Get("name"),
+			)
+	}
+
+	d.Set("project", stringParts[1])
```

Comment:
Usually you only need to set the id of a resource and the subsequent get should be able to read the rest. If the 'name' is the equivalent of the self_link then you should be able to just set the id to the name without needing to parse the project from the string.